tutorial/input_output.py

```diff
@@ -0,0 +1,234 @@
+from .common import Question, Quiz
+
+
+class StringOutput(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="Can the function <code>print()</code> be used on strings only?",
```

Comment:
Better write it as a normal sentence, not a question:
```suggestion
            question="The function <code>print()</code> can be used only to output strings.",
```